### PR TITLE
Create configuration of project, add Eslint, EditorConfig and Travis

### PR DESCRIPTION
**Pull request by Zitle-Nancy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,21 @@
   "dependencies": {
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
+    "react-screen-orientation": "0.0.3",
```
Comment (ivandevp) [inline package.json]: Remove this dependency since we don't need it.

File: src/components/snake.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.snake{
```
Comment (ivandevp) [inline src/components/snake.css]: Let's add a space before open curly bracket (`{`), please 🙏

File: src/components/snake.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React,{ Component } from 'react';
+import './snake.css';
+
+export default class Snake extends Component{
+  render(){
+    return(
+      <div className="snake" />
+    )
+  }
+}
```
Comment (ivandevp) [inline src/components/snake.jsx]: Linter check is missing here 🤔

File: src/registerServiceWorker.jsx [hunk 4]
```diff
@@ -8,6 +8,8 @@
 // To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
 // This link also includes instructions on opting out of this behavior.
 
+/** window */
+/* eslint operator-linebreak: ["error", "after"] */
```
Comment (ivandevp) [inline src/registerServiceWorker.jsx]: Is necessary that this file should have a `.jsx` extension?

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,jsx}]
+charset = utf-8
```
Comment (ivandevp) [inline .editorconfig:11]: Should this config affect CSS, too? 🤔

**Review by ivandevp — CHANGES_REQUESTED** [review]
Awesome!! 🌮 🎉 

I have left some comments in order to merge this PR!!! Great job though.

File: src/components/snake.css (hunk 2 above)
Comment (Zitle-Nancy) [reply src/components/snake.css]: Why you need to add a space before open curly bracket? 🤔

Comment (ivandevp) [reply src/components/snake.css]: If we follow linter rules for CSS like [`stylelint`](https://stylelint.io), you will see that spaces are good options for writing css rules. You can paste your code and try it [here](https://stylelint.io/demo/).

Comment (ivandevp) [reply src/components/snake.css]: Actually, I think we should add a linter for CSS 🤔

**Review by ivandevp — APPROVED** [review]
Cool!!! We're ready to merge now!!! 

Great!

**Pull request merged**